antarest/common/logger.py

```diff
@@ -0,0 +1,27 @@
+import logging
+from dataclasses import dataclass
+from typing import Optional
+
+from dataclasses_json.api import dataclass_json
+
+from antarest.common.config import Config, register_config
+
+
+@dataclass_json
+@dataclass
+class LoggingConfig:
```

Comment:
Why we separate config ? We don't want to centralize config inside one object ?

Reply:
at the very first I thought so, but as I like to think we could easily split the code in several micro services, I splitted the config in a plugin-like way